www/docs/client/links/httpSubscriptionLink.md

```diff
@@ -113,6 +115,12 @@ const trpc = createTRPCClient<AppRouter>({
             },
           }; // you either need to typecast to `EventSourceInit` or use `as any` or override the types by a `declare global` statement
         },
+
+        // Optional: EventSource will automatically recover from disconnections, but always uses the initial headers
+        // If your auth header expires, you can force a restart and fresh eventSourceOptions() call when shouldRecreateOnError() returns true
+        shouldRecreateOnError(status, _error) {
+          return opts.type === 'http-error' && [401, 403].includes(opts.status);
+        },
```

Comment:
I think this is worth its own header in the docs as well

Reply:
Fair point, done and cleaned up this one since it becomes just noise